app.js

```diff
@@ -0,0 +1,84 @@
+var path = require('path');
+var fs = require('fs');
+var express = require('express');
+var Router = require('react-router');
+
+var indexStaticWatcher = require('./lib/index-static-watcher').create();
+var PORT = process.env.PORT || 8008;
+var PRODUCTION = (process.env.NODE_ENV === 'production');
+var DIST_DIR = path.join(__dirname, 'dist');
+
+var indexStatic;
+var router;
+var app = express();
+
+var startProdApp = function() {
+  console.log([
+    'Production mode enabled. Note that "npm install" is assumed to',
+    'have recently been run with NODE_ENV="production". If this is not',
+    'the case, some or all static assets may be out of date.'
+  ].join('\n'));
+  indexStaticWatcher.build(function(err, newIndexStatic) {
+    if (err) {
+      throw err;
+    }
+
+    console.log('Built server-side bundle.');
```

Comment:
(from [this comment](https://github.com/mozilla/teach.webmaker.org/pull/936#discussion_r31315876))
>Er, another clarification: the dynamic lightweight server **does not serve any .html files in the `dist/` directory**. In fact, when deploying to Heroku, `npm install` also generates the `dist/` directory via a postinstall step, and you'll note that it doesn't actually generate the static HTML files--only the LESS/CSS, client-side JS, and so on.

> Even if there happen to be any leftover `.html` files in the `dist/` directory on your development machine when you run the dynamic lightweight server, they're never actually served--the server's router intercepts all requests to URLs like `/clubs/` and dynamically generates the responses before the [express static middleware](http://expressjs.com/starter/static-files.html) ever sees them.

I tried running `NODE_ENV="production" ORIGIN="http://localhost:8008" node app.js` --> made some copy changes on the homepage --> refreshed the page.  However, the changes I just made didn't get reflected on the site.  If I ran `NODE_ENV="production" ORIGIN="http://localhost:8008" npm install` after I made the copy changes to `home.jsx` and then ran `NODE_ENV="production" ORIGIN="http://localhost:8008" node app.js` again I would see the changes showing on the site.  
If "the server's router intercepts all requests to URLs like `/clubs/` and dynamically generates the responses before the [express static middleware](http://expressjs.com/starter/static-files.html) ever sees them" I would assume I wouldn't have to run `npm install` to copy my code changes?